package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "nodejs-school",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon --exec babel-node --presets env,stage-2 app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/EugeneAlexeenko/nodejs-school.git"
+  },
+  "author": "Eugene Alexeenko",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/EugeneAlexeenko/nodejs-school/issues"
+  },
+  "homepage": "https://github.com/EugeneAlexeenko/nodejs-school#readme",
+  "devDependencies": {
+    "babel-cli": "6.26.0",
```

Comment:
```babel-cli``` wasn't in the requirements, but I needed ```babel-node``` to compile project. Is there is another solution?